### DISCUSSION TITLE
How can I upload my GPU benchmark results?

### DISCUSSION THREAD

**Discussion by ghost** [pytorch/benchmark#2088] [General]
I want to publish results for the AMD 7900 XT GPU (supported from November 2023 by AMD in the PyTorch ROCm version).
Trying to create a new branch to make a PR, but get "fatal: Authentication failed for 'https://github.com/ryujaehun/pytorch-gpu-benchmark.git/".
How should I push my benchmark results?

**Comment by xuzhao9** [comment]
May I ask what command did you run to create a new branch to make a PR? As far as I know, pytorch/benchmark is not using ryujaehun/pytorch-gpu-benchmark.
